store/generated_key/memstore/memstore.go

```diff
@@ -3,52 +3,48 @@ package memstore
 import (
 	"context"
 	"crypto/rand"
-	"errors"
 	"fmt"
 	"math/big"
-	"sync"
-	"time"
 
 	"github.com/ethereum/go-ethereum/rlp"
 
 	"github.com/Layr-Labs/eigenda-proxy/common"
+	"github.com/Layr-Labs/eigenda-proxy/store/generated_key/memstore/ephemeral_db"
 	"github.com/Layr-Labs/eigenda-proxy/store/generated_key/memstore/memconfig"
 	"github.com/Layr-Labs/eigenda-proxy/verify/v1"
-	"github.com/Layr-Labs/eigenda/api"
 	"github.com/Layr-Labs/eigenda/api/clients/codecs"
 	eigenda_common "github.com/Layr-Labs/eigenda/api/grpc/common"
 	"github.com/Layr-Labs/eigenda/api/grpc/disperser"
 	"github.com/Layr-Labs/eigensdk-go/logging"
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
+func randomBytes(size uint) []byte {
+	entropy := make([]byte, size)
+	_, _ = rand.Read(entropy)
+	return entropy
+}
+
 const (
-	DefaultPruneInterval = 500 * time.Millisecond
 	BytesPerFieldElement = 32
 )
 
 /*
 MemStore is a simple in-memory store for blobs which uses an expiration
 time to evict blobs to best emulate the ephemeral nature of blobs dispersed to
-EigenDA operators.
+EigenDA V1 operators.
 */
 type MemStore struct {
+	*ephemeral_db.DB
```

Comment:
```suggestion
        // DB stores a mapping of EigenDACertV1 -> Blob
	*ephemeral_db.DB
```

Reply:
Actually just realized we use the InclusionProof and not the cert as key... can we just use the cert? feels more canonical. Otherwise, can we document why we use the InclusionProof (it says b/c eigenda does so, but does that matter?)